RELEASING.md

```diff
@@ -200,9 +208,9 @@ at the same time that subcomponent release branches are cut, often well before t
 
 1. On netlify create a new site using the `release-vX.Y` branch (You should at least have write access to this repo for site creation) 
 
-1. Rename the randomly generated site name to follow the same naming convention as other releases.
+1. Rename the randomly generated site name to follow the same naming convention as other releases (Ex: `calico-VX-Y`).
 
-1. Ensure that the site is generated properly by visiting site URL (Ex. https://<site-name>.netlify.app/archive/vX.Y/).  
+1. Ensure that the site is generated properly by visiting site URL (Ex. https://calico-VX-Y.netlify.app/archive/vX.Y/).  
```

Comment:
nitpick, but can you use a lowercase v e.g. `vX.Y` instead of `VX.Y`
```suggestion
1. Ensure that the site is generated properly by visiting site URL (Ex. https://calico-vX-Y.netlify.app/archive/vX.Y/).  
```

Reply:
changes throughout the doc in the new push